components/feedback/dialog/AriaDialogTitle.tsx

```diff
@@ -0,0 +1,57 @@
+import Box from "@mui/material/Box";
+import DialogTitle from "@mui/material/DialogTitle";
+import IconButton from "@mui/material/IconButton";
+import Stack from "@mui/material/Stack";
+import SvgIcon from "@mui/material/SvgIcon";
+import Typography from "@mui/material/Typography";
```

Comment:
I think we can import it in a better way 🤔 

```suggestion
import { Box, DialogTitle, IconButton, Stack, SvgIcon, Typography } from "@mui/material"
```

Reply:
This is debatable, but I prefer Altair's approach. The reason is about tree-shaking -> https://www.telerik.com/blogs/tree-shaking-basics-for-react-applications